core/src/gl/texture.cpp

```diff
@@ -8,6 +8,14 @@
 #include "tangram.h"
 #include "log.h"
 
+// Only JPEG, PNG and GIF
+#define STBI_NO_BMP
+#define STBI_NO_PSD
+#define STBI_NO_TGA
```

Comment:
Yes I agree with that. But this can be a ES only thing. Users of ES may not be users of JS.